platforms/android/library/src/main/java/io/element/android/wysiwyg/utils/ViewLazyViewModelExtension.kt

```diff
@@ -1,19 +1,29 @@
 package io.element.android.wysiwyg.utils
 
+import android.content.Context
+import android.content.ContextWrapper
 import android.view.View
 import androidx.annotation.MainThread
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelLazy
 import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelStoreOwner
 import androidx.lifecycle.findViewTreeViewModelStoreOwner
 
 @MainThread
 inline fun <reified VM : ViewModel> View.viewModel(
-    noinline viewModelInitializer: (() -> VM)? = null
+    useViewTreeOwner: Boolean = false,
+    noinline viewModelInitializer: (() -> VM)? = null,
 ): Lazy<VM> {
     return ViewModelLazy(
         viewModelClass = VM::class,
-        storeProducer = { findViewTreeViewModelStoreOwner()!!.viewModelStore },
+        storeProducer = {
+            if (useViewTreeOwner) {
+                findViewTreeViewModelStoreOwner()!!.viewModelStore
+            } else {
+                (getBaseContext(context) as ViewModelStoreOwner).viewModelStore
```

Comment:
Add a new (default) way of getting the `ViewModel` that won't crash when using Compose and having config changes (like screen rotation).

@jonnyandrew I see few other alternatives, but if you do find one please modify this code. Also, feel free to remove the old code if you think it's not worth keeping it around.